tests/components/airthings/test_sensor.py

```diff
@@ -0,0 +1,158 @@
+"""Test the Airthings sensors."""
+
+from unittest.mock import AsyncMock, patch
+
+from airthings import AirthingsDevice
+
+from homeassistant.components.airthings import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from . import TEST_DATA
+
+from tests.common import MockConfigEntry
+
+
+async def init_with_device(hass: HomeAssistant, device: AirthingsDevice) -> None:
+    """Set up the Airthings integration in Home Assistant with a single device."""
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data=TEST_DATA,
+    )
+
+    mock_airthings = AsyncMock()
+    mock_airthings.update_devices.return_value = {device.device_id: device}
+
+    with (
+        patch(
+            "airthings.get_token",
+            return_value="test_token",
+        ),
+        patch(
+            "homeassistant.components.airthings.Airthings",
+            return_value=mock_airthings,
+        ),
+    ):
+        entry.add_to_hass(hass)
+        await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
```

Comment:
this part can be part of `__init__.py`. I generally name these function `setup_integration`